src/inverseRobot.js

```diff
@@ -24,7 +24,17 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const inverseRobotObject = {};
+
+  for (const key in robot) {
+    if (inverseRobotObject[robot[key]] in robot) {
+      return null;
+    }
```

Comment:
You should check if this property exists in inversedRobot, not in robot:
```suggestion
    if (inverseRobotObject[robot[key]]) {
      return null;
    }
```